views/jsconfig.handlebars

```diff
@@ -1,8 +1,16 @@
+{{!-- This file should be es5 only --}}
+var isIE = navigator.userAgent.toLowerCase().indexOf('msie') > -1 ||
+    !!navigator.userAgent.toLowerCase().match(/trident\/7\./);
+var isUnsupportedPage = location.pathname.includes('/unsupported');
```

Comment:
I don't think we can use `.includes()` here, per https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes#Browser_compatibility

May have to just use a cheap `.indexOf('/unsupported') !== -1)` or similar.